appveyor.yml

```diff
@@ -7,14 +7,22 @@ environment:
   global:
     DOWNLOAD_DIR: _download_cache
     APPVEYOR_SAVE_CACHE_ON_ERROR: true
-    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd
+    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd;C:\Program Files\7-Zip
```

Comment:
why do we need 7zip? Is that something that comes by default? If not we can use the tar that we have from msys2

Reply:
It's part of the default appveyor install and used below in build_script. Sure, will switch to msys2